scripts/combine_lightcurves.py

```diff
@@ -1,43 +1,93 @@
+#!/usr/bin/env python3
+
 import os
+import shutil
 import json
 import argparse
+from json.decoder import JSONDecodeError
 
-import glob
+from pocs.utils.google.storage import PanStorage
 
 
 def get_curves_for_pic(pic):
+    """Read light curve files and add all curves to array.
+
+    :param pic: the PIC id for the star to make a master light curve for
+    :return: an array of all the light curves for the given PIC
+    """
     curves = []
-    for filename in glob.iglob("data/LC/{}/**/*.json".format(pic), recursive=True):
-        with open(filename, 'r') as f:
-            c = json.load(f)
-            curves.append(c)
-    return curves
+    bucket_name = 'panoptes-simulated-data'
+    prefix = "LC/{}/".format(pic)
+    temp_dir = '{}/temp'.format(os.getenv('PANDIR', default='/var/panoptes'))
```

Comment:
Also, you're doing an rmtree below on the temp folder, and it has a pretty generic name.  I know we assume that only one process will run on a machine at a time, but this could still be dangerous.

Consider using a default temp folder that is unlikely to collide, something like /tmp/plc-combine-<random number>    If you want to be extra paranoid, you can even check that it does not exist before you start using it :)